packages/swingset-runner/test/test-demo.js

```diff
@@ -40,6 +40,7 @@ async function innerTest(t, extraFlags, dbdir) {
       output += data;
     });
     proc.addListener('exit', code => {
+      t.log(output);
```

Comment:
@FUDCo I made this change to help debug the swingset tests.

Basically, `t.log` doesn't produce any output unless you run `yarn test -v`.  Is that acceptable to you, or should I comment this line out?

Reply:
In this case the test is watching the output from the vat console, but there's no need for that vat console output to appear in the output of the test itself, so the `t.log` make be unnecessary.  But it does no harm.

Reply:
Thanks.